keyboards/nullbitsco/nibble/keymaps/snailmap/keymap.c

```diff
@@ -437,17 +428,16 @@ static void render_wpm_graph(int current_wpm) {
 
 // Call OLED functions
 void oled_task_user(void) {
-    // Fix layer names, draw keyboard, prevent redraw
+    // Draw OLED keyboard, prevent redraw
     if (first_loop) {
-        shorten_layer_names();
```

Comment:
For this, if you want the shorten named, you could add additional defines, and then pick the set of defines based on that. 

Eg: 
```c
    oled_write_P(PSTR(oled_horizontal ? MA_LAYER_NAME_SHORT : MA_LAYER_NAME), false);
```

Reply:
Hmm, neat idea. But I think some user's may want to change the layer names, and having two sets might cause confusion.
That function wasn't very useful anyway. If the layer names are being drawn over other parts of the display, it should be fairly clear there are too many characters.

Reply:
Fair enough.  I figured it was worth mentioning, at least.